tests/providers/google/cloud/hooks/test_cloud_sql.py

```diff
@@ -1143,7 +1143,7 @@ def test_cloudsql_database_hook_get_database_hook(self, get_connection):
 
 class TestCloudSqlDatabaseHook(unittest.TestCase):
 
-    @mock.patch('airflow.contrib.hooks.gcp_sql_hook.CloudSQLDatabaseHook.get_connection')
+    @mock.patch('airflow.providers.google.cloud.hooks.cloud_sql.CloudSQLDatabaseHook.get_connection')
```

Comment:
How this change is related to PR title?

Reply:
Thanks for pointing this out, it will help me improve.
After running the test, I saw deprecation warning and decided to fix it. 
I know now that I should have created an issue for it or make a better PR title.
I really appreciate 🙂

Reply:
+1  It looks good. We should delete references to airflow.contrib.